src/Arbitrary.hs

```diff
@@ -49,26 +57,36 @@ instance Arbitrary PageTopicOverviewResultPhase where
 instance Arbitrary PageTopicOverviewDelegations where
     arbitrary = pure PageTopicOverviewDelegations
 
+instance Arbitrary PageIdeaDetail where
+    arbitrary = oneof
+        [ PageIdeaDetailNewIdeas'            <$> arb
+        , PageIdeaDetailRefinementPhase'     <$> arb
+        , PageIdeaDetailJuryPhase'           <$> arb
+        , PageIdeaDetailVotingPhase'         <$> arb
+        , PageIdeaDetailFeasibleNotFeasible' <$> arb
```

Comment:
I don't understand `PageIdeaDetailFeasibleNotFeasible`.  How much of the business logic should we pull into haddocks, and where?